networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/nsxv3_facada.py

```diff
@@ -177,33 +153,34 @@ def get_switch_id_for_segmentation_id(self, segmentation_id):
             return ls.id
 
     def get_port(self, sdk_service, sdk_model):
-        svc = sdk_service(self.stub_config)
+        sdk_service(self.stub_config)
         sdk_type = str(sdk_model.__class__.__name__)
         attr_key = "attachment.id"
         attr_val = str(sdk_model.attachment["id"])
         msg = "Getting '{}'  {}='{}' ... ".format(sdk_type, attr_key, attr_val)
         LOG.info(msg)
 
         kwargs = {
-            "sdk_service": sdk_service, 
-            "sdk_model": sdk_model, 
-            "attr_key": "attachment_id", 
+            "sdk_service": sdk_service,
+            "sdk_model": sdk_model,
+            "attr_key": "attachment_id",
             "attr_val": attr_val
         }
 
-        return self.retry_until_result(self.get_by_attr, kwargs=kwargs, 
-            retry_max=3, retry_sleep=5)
+        return self.retry_until_result(self.get_by_attr, kwargs=kwargs,
+                                       retry_max=3, retry_sleep=5)
```

Comment:
If you go the way you did with adding global variables, you should either import them for these 2 calls or change your way by adding a default to `retry_until_result()`.